modules/standard/LinkedLists.chpl

```diff
@@ -203,6 +203,43 @@ record LinkedList {
      return ret;
    }
 
+  /*
+    find `e` in the list.
+    returns true if item is found, otherwise returns false
+   */
+  proc search(e : eltType): bool {
+     for item in this {
+       if (e == item) {
+         return true;
+       }
+     }
+    return false;
+  }
+  
+  /*
+    Returns the data stored in the first element of the list
+    It is an error to call front() on an empty list
+   */
+  proc front(): eltType throws {
```

Comment:
I'm not sure should we return eltType instead of ref. Anyway, similar procedures of List return ref.